hello.go

```diff
@@ -42,12 +45,40 @@ func main() {
 	res, err := client.Do(req)
 	if err == nil {
 		defer res.Body.Close()
-		responseBody, err := ioutil.ReadAll(res.Body)
-		if err != nil {
-			log.Fatal(err)
+		if res.StatusCode == 200 {
+			responseBody, err := ioutil.ReadAll(res.Body)
+			if err != nil {
+				log.Fatal(err)
+			}
+			project = string(responseBody)
+			if project != "" {
+				projectFound = true
+			}
+		}
+	}
+
+	// Get region from metadata server
+	region := "???"
+	regionFound := false
+	req, _ = http.NewRequest("GET", "http://metadata.google.internal/computeMetadata/v1/instance/region", nil)
+	req.Header.Set("Metadata-Flavor", "Google")
+	res, err = client.Do(req)
+	if err == nil {
+		defer res.Body.Close()
+		if res.StatusCode == 200 {
+			log.Print("region")
+			log.Print(res.StatusCode)
+			responseBody, err := ioutil.ReadAll(res.Body)
+			if err != nil {
+				log.Fatal(err)
+			}
+			re := regexp.MustCompile(`regions\/(.*)$`)
```

Comment:
This seems overly complex. Why can’t we just trim the regions/ prefix?

Reply:
the string is something like `projects/<number>/regions/<region>`.
So I need some kind of regex.